### ISSUE TITLE
HttpClientModule dependency, otherwise NullInjectorError: No provider for HttpBackend! 

### ISSUE THREAD

**Issue by markterrill** [dbfannin/ngx-logger#211]
Summary, installing the module per docs results in this error unless you install HttpClientModule. If you're not using server logging, it's set to OFF and no server logging URI, then surely you shouldn't need to include this dependency.

Error:
```NullInjectorError: R3InjectorError(RecentCooksPageModule)[NGXLogger -> NGXLogger -> NGXMapperService -> HttpBackend -> HttpBackend -> HttpBackend]: NullInjectorError: No provider for HttpBackend! ```

Current fix in app.module.ts:
```
import { HttpClientModule } from "@angular/common/http";

@NgModule({
  declarations: [
    AppComponent
  ],
  entryComponents: [],
  imports: [
    BrowserModule,
    AppRoutingModule,
    HttpClientModule,
    LoggerModule.forRoot(
        {
          serverLogLevel: NgxLoggerLevel.OFF,

          level: NgxLoggerLevel.DEBUG,
        }
    ),
  ],
.........
```

I'll submit a PR doc edit on the readme

**Comment by MickL** [comment]
What if I want to use the logger but I dont want to include the HttpClientModule?

**Comment by MickL** [comment]
I would also love to have ngx-logger not require an HttpClientModule to be imported. I am working on libraries that are not using the http client but ngx logger requires it to be imported.

@qortex @dbfannin This is marked as a bug. Is it? Any could you address this anytime soon? :)

**Comment by bmtheo** [comment]
This is not a bug, it is a feature request, I approved and merged the #212 PR to update the doc

I am closing this issue as there is already one existing for removing this dep (see #134)

**Comment by bmtheo** [comment]
#134 just closed, HttpClientModule import is now optional :)
@markterrill @MickL
